lib/sycamore/sycamore/transforms/detr_partitioner.py

```diff
@@ -766,15 +769,14 @@ def extract_ocr(
     elements: list[list[Element]],
     ocr_images: bool = False,
     ocr_model: Union[str, OcrModel] = "easyocr",
+    text_extraction_options: dict[str, Any] = {},
 ) -> list[list[Element]]:
     ocr_model_obj: OcrModel
     if isinstance(ocr_model, OcrModel):
         ocr_model_obj = ocr_model
     else:
-        model_cls = EXTRACTOR_DICT.get(ocr_model)
-        if not model_cls:
-            raise ValueError(f"Unknown OCR Model: {ocr_model}")
-        ocr_model_obj = model_cls()
+        ocr_model_obj = cast(OcrModel, get_text_extractor(ocr_model, **text_extraction_options))
```

Comment:
Do we want to assert instead of `cast`? We can't handle a non `OcrModel`

Reply:
Yeah, let me fix this.